locales/locales/en@lolcat.json

```diff
@@ -5818,5 +5818,13 @@
     "commentNavigatorShowByDefaultTitle": {
         "message": "Show By Default",
         "description": ""
+    },
```

Comment:
Thanks for due diligence, but you only need to edit `en.json`. All the other languages will copy `en.json`'s values into they get translations hooked up.